File: Models/PersonalDataModel.cs
namespace SistemaElectoral.Models;

public class PersonalData
{
    public int IdPersonalData { get; set; }
    public string Dni { get; set; } = "";
    public string Name { get; set; } = "";
    public bool Gender { get; set; }
    public string Phone { get; set; } = "";
}

File: server/src/models/personalData.model.js
class PersonalData {
    constructor({ idPersonalData, dni, name, gender, phone }) {
        this.idPersonalData = idPersonalData;
        this.dni = dni;
        this.name = name;
        this.gender = gender ? !!gender : undefined;
        this.phone = phone;
    }
}

module.exports = PersonalData;